File: engine/engine-model/src/main/scala/tools/packet/showitemgaininchat/ShowMakerEffect.scala
package tools.packet.showitemgaininchat

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowMakerEffect(private var _success: Boolean) extends PacketInput {
  def success: Boolean = _success

  override def opcode(): SendOpcode = SendOpcode.SHOW_ITEM_GAIN_IN_CHAT
}

File: engine/engine-model/src/main/java/tools/packet/showitemgaininchat/ShowMakerEffect.java
package tools.packet.showitemgaininchat;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowMakerEffect(Boolean success) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_ITEM_GAIN_IN_CHAT;
   }
}